quickjs.c

```diff
@@ -26267,6 +26267,29 @@ static char *js_default_module_normalize_name(JSContext *ctx,
     int len;
 
     if (name[0] != '.') {
+        /* Backwards compatibility for stdlib rename. */
+        static char *prefix = "qjs:";
+        static char *names[] = {
+            "bjson",
+            "std",
+            "os",
+            NULL
+        };
+        int i;
+        for (i = 0; names[i]; i++) {
+            if (!strcmp(name, names[i])) {
+#ifndef NDEBUG
+                printf("WARN: Standard library modules should be prefixed with `qjs:`. Example: qjs:%s\n", name);
+#endif
+                len = strlen(prefix) + strlen(name) + 1;
+                filename = js_mallocz(ctx, len);
+                if (!filename)
+                    return NULL;
+                pstrcat(filename, len, prefix);
+                pstrcat(filename, len, name);
+                return filename;
+            }
+        }
```

Comment:
Instead of strcat'ing stuff together, can I suggest this?
```suggestion
        static const char names[] = "qjs:bjson\0qjs:std\0qjs:os";
        for (const char *p = names; p < endof(names); p += 1 + strlen(p)) {
            if (!strcmp(name, p+4)) {
#ifndef NDEBUG
                printf("WARN: Standard library modules should be prefixed with `qjs:`. Example: %s\n", p);
#endif
                return js_strdup(ctx, p);
            }
        }
```

Reply:
Nice! 👍